src/ConstructiveSolidGeometry/plotting/SurfacePrimitives/ConeMantle.jl

```diff
@@ -1,17 +1,52 @@
-@recipe function f(cm::ConeMantle, n = 40; subn = 10)
-    ls = lines(cm)
-    linecolor --> :black
-    @series begin
-        label --> "Cone Mantle"
-        ls[1]
+function mesh(cm::ConeMantle{T}; n = 30)::Mesh{T} where {T}
+    rbot, rtop = get_r_limits(cm)
+    φMin, φMax = get_φ_limits(cm)
+    
+    f = (φMax - φMin)/(2π)
+    n = Int(ceil(n*f))
+    φ = range(φMin, φMax, length = n+1)
+    z = range(-cm.hZ, cm.hZ, length = 2)
+
+    X::Array{T,2} = [radius_at_z(cm,z[j])*cos(φ_i) for φ_i in φ, j in 1:length(z)]
+    Y::Array{T,2} = [radius_at_z(cm,z[j])*sin(φ_i) for φ_i in φ, j in 1:length(z)]
```

Comment:
Instead of `j in 1:length(z)` and calling `radius_at_z(cm, z[j])`,
you could do `zj in z` and `radius_at_z(cm, zj)`